src/bootstrap-table.js

```diff
@@ -810,7 +810,8 @@
                 .off('click').on('click', function () {
                     that.options.cardView = !that.options.cardView;
                     that.initHeader();
-                    that.initToolbar();
```

Comment:
Hi, again !!

I've found another error. In "BootstrapTable.prototype.initBody = function (fixedScroll) { ..." It's necessary add the styles of checkbox header to your cell so I've changed this line:
`<div class="card-view">' : sprintf('<td class="bs-checkbox" %s>',style), // Add styles to checkbox cell.`

.
![captura](https://cloud.githubusercontent.com/assets/8124520/7234740/783f43e4-e789-11e4-8879-9ee058dffbc8.PNG)